wfl/configset.py

```diff
@@ -50,8 +50,8 @@ class ConfigSet_in:
         path to prepend to every file name
     input_files: str / iterable(str) / 2-tuple(str) / iterable(2-tuple(str))
         Input file name globs, optionally as 2-tuples of (filename_glob, index).
-        The latter only works with more than one tuple
-        index here overrides separate default_index arg
+        The latter only works with more than one tuple index here overrides 
```

Comment:
line breaks were meaningful before.  Need "." after "one tuple".

Reply:
of course; reverted back.